embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/config/behavior/SdkEndpointBehavior.kt

```diff
@@ -23,10 +23,10 @@ internal class SdkEndpointBehavior(
     /**
      * Data base URL.
      */
-    fun getData(appId: String): String = local?.data ?: "https://a-$appId.$DATA_DEFAULT"
+    fun getData(appId: String?): String = local?.data ?: "https://a-$appId.$DATA_DEFAULT"
```

Comment:
i think it would be better if this returned an empty string if `appId` is not set

Reply:
Updated to return empty string